libshvchainpack/include/shv/chainpack/metamethod.h

```diff
@@ -92,42 +95,42 @@ class SHVCHAINPACK_DECL_EXPORT MetaMethod
 	static Signature signatureFromString(const std::string &sigstr);
 	static const char* signatureToString(Signature sig);
 	static std::string flagsToString(unsigned flags);
-	static const char* accessLevelToString(AccessLevel access_level);
+	static const char* accessLevelToAccessString(AccessLevel access_level);
+	static std::optional<AccessLevel> accessLevelFromAccessString(std::string_view s);
+	static std::optional<AccessLevel> accessLevelFromInt(int i);
+	//static AccessLevel findAccessLevel(const std::string &access);
 private:
 	std::string m_name;
 	unsigned m_flags = 0;
 	std::string m_param;
 	std::string m_result;
-	AccessLevel m_accessLevel;
+	int m_accessLevelInt;
```

Comment:
same as above, can't this be `AccessLevel` directly?

Reply:
There is also `AccessLevel accessLeve() const` member function.